jwst/pipeline/calwebb_spec2.py

```diff
@@ -27,6 +27,9 @@
 
 __all__ = ['Spec2Pipeline']
 
+# Classify various exposure types.
+WFSS_TYPES = ["NIS_WFSS", "NRC_WFSS"]
```

Comment:
Might want to add "NRC_GRISM" to this list just for backwards compatibility (it's the old form of NRC_WFSS).